backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -174,38 +175,37 @@ public static void main(String[] args) {
       // --------------- Start reading properties --------------------------
       App app = App.getInstance();
       Config config = Config.getInstance();
-      // --------------- End reading properties --------------------------
 
       // Initialize database configuration and maybe run migration
-      boolean liquibaseMigration =
-          Boolean.parseBoolean(
-              Optional.ofNullable(System.getenv(ModelDBConstants.LIQUIBASE_MIGRATION))
-                  .orElse("false"));
-      if (liquibaseMigration) {
-        LOGGER.info("Liquibase migration starting");
-        ModelDBHibernateUtil.runLiquibaseMigration(config.database);
-        LOGGER.info("Liquibase migration done");
-
-        ModelDBHibernateUtil.createOrGetSessionFactory(config.database);
-
-        LOGGER.info("Code migration starting");
-        ModelDBHibernateUtil.runMigration(config.database);
-        LOGGER.info("Code migration done");
-
-        boolean runLiquibaseSeparate =
-            Boolean.parseBoolean(
-                Optional.ofNullable(System.getenv(ModelDBConstants.RUN_LIQUIBASE_SEPARATE))
-                    .orElse("false"));
-        if (runLiquibaseSeparate) {
-          return;
-        }
-      }
+      if (migrate(config)) return;
 
-      ModelDBHibernateUtil.createOrGetSessionFactory(config.database);
+      // Configure server
+      System.getProperties().put("server.port", config.springServer.port);
+
+      app.shutdownTimeout = config.springServer.shutdownTimeout;
 
-      // --------------- Start Initialize modelDB gRPC server --------------------------
+      // Initialize services that we depend on
+      ServiceSet services = ServiceSet.fromConfig(config);
+
+      // Initialize data access
+      DAOSet daos = DAOSet.fromServices(services);
+
+      // Initialize telemetry
+      initializeTelemetryBasedOnConfig(config);
+
+      // Initialize cron jobs
+      CronJobUtils.initializeBasedOnConfig(
+          config, services.authService, services.roleService, services.artifactStoreService);
```

Comment:
it should be `initializeCronJobs` instead of `initializeBasedOnConfig`

Reply:
Could you suggest the change via github? It's faster to fix that way =) I can open the IDE if too hard

Reply:
Done, Sorry for that.